src/Sarif/Visitors/SarifVersionOneToCurrentVisitor.cs

```diff
@@ -316,19 +303,19 @@ internal string GetLogicalLocationName(string fullyQualifiedLogicalName)
                 throw new ArgumentNullException(nameof(fullyQualifiedLogicalName));
             }
 
-            return fullyQualifiedLogicalName.Split(Utilities.FullyQualifiedNameDelimiters,
+            return fullyQualifiedLogicalName.Split(Utilities.DefaultFullyQualifiedNameDelimiters,
                                                    StringSplitOptions.RemoveEmptyEntries).Last();
         }
 
-        internal void FixUpAllLogicalLocations()
+        internal void RemoveRedundantLogicalLocationProperties()
         {
             foreach (string key in _currentRun.LogicalLocations.Keys)
             {
-                FixUpLogicalLocation(key);
+                RemoveRedundantProperties(key);
```

Comment:
Should I be concerned that `RemoveRedundantProperties` is called twice, once from `RemoveRedundantLogicalLocationProperties` and once from `AddLogicalLocation`?

Reply:
No. The call to RemoveRedundantLogicalLocationProperties is made after the v1 logicalLocation dictionary is imported, to ensure the properties that should be null are. The second call is made for each new l.L.

Reply:
Got it.